### PR TITLE
BAZ-17/searchBar

### PR DESCRIPTION
**Pull request by BaburDilshat**
BAZ-17/searchBar
Link to Story
https://re-boot.atlassian.net/browse/BAZ-17

Description
Worked on search bar

Screenshot(s) / GIF(s):
image

Type of change
 Bug fix
 New feature
 Dependency / Configuration update
 Documentation update
Checklist:
 My code follows the project's style guidelines
 I have created a Storybook story for my component(s)
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 I have made corresponding changes to the documentation
 My changes generate no new warnings
 I have added tests that prove my fix is effective or that my feature works
 New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Jack-Vu — COMMENTED** [review]
Looks pretty good so far, but you might need to handle the enter key since it is an input field

File: src/app/_components/SearchBar/DesktopSearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { MagnifyingGlassIcon } from "@heroicons/react/24/solid";
+
+const DesktopSearchBar = () => {
+  const onSubmit = () => {
+    alert("Submitting from modal");
+  };
+  const handlerFuntion = () => {
```
Comment (Jack-Vu) [inline src/app/_components/SearchBar/DesktopSearchBar.tsx]: this function and onSubmit do the same thing

File: src/app/_components/SearchBar/MobileSearchBar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import type { Meta, StoryObj } from "@storybook/react";
+// import { Sample } from "./MobileSearchBar";
```
Comment (Jack-Vu) [inline src/app/_components/SearchBar/MobileSearchBar.stories.tsx]: remove this comment

**Review by Jack-Vu — CHANGES_REQUESTED** [review]
A few changes needed

**Review by Jack-Vu — COMMENTED** [review]
Also need to resolve conflict with dev branch

**Review by Muradil-Erkin — DISMISSED** [review]

**Review by Muradil-Erkin — APPROVED** [review]

**Pull request merged**